mmv1/templates/terraform/examples/eventarc-workflows.tf.erb

```diff
@@ -0,0 +1,108 @@
+# [START eventarc_terraform_enableapis]
```

Comment:
this file can be removed in place of the new `eventarc_workflows.tf.erb` file

Reply:
Done, thank you Megan.